packages/components/src/components/tabs/tabs.jsx

```diff
@@ -98,54 +104,69 @@ const Tabs = ({
     return (
         <div
             className={classNames('dc-tabs', {
-                [`dc-tabs dc-tabs--${className}`]: className,
+                [`dc-tabs--${className}`]: className,
```

Comment:
Will this change affect usages anywhere else?

Reply:
@aaron-binary that's extra, we also have it in the above line.